diesel_cli/src/cli.rs

```diff
@@ -57,6 +57,9 @@ pub fn build_cli() -> App<'static, 'static> {
         ).subcommand(SubCommand::with_name("drop")
             .about("Drops the database specified in your DATABASE_URL.")
             .setting(AppSettings::Hidden)
+        ).subcommand(SubCommand::with_name("touch")
```

Comment:
Yeah, the issue will hopefully disappear when the examples stop overlapping (the other thread).